src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -53,6 +54,14 @@ function setHasLocationChangingListeners(hasListeners: boolean) {
   hasLocationChangingEventListeners = hasListeners;
 }
 
+function scrollToElement(id : string) : void {
```

Comment:
This only accounts for `<elem id="...">` but would also need to account for `<a name="...">` (https://www.tagindex.net/html/link/a_name.html).

The parameter should be an anchor name, not an element ID. I'm not sure whether it should prioritize elements by ID over anchors by name - would it be possible to check how native `<a>` tags work and follow the same prioritization?